tutorials/classical_simulation/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,565 @@
+{
```

Comment:
<p>eventually href the other notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>